src/Components/Results/ResultsError/ResultsError.tsx

```diff
@@ -18,8 +20,9 @@ const ResultsError = () => {
         <p className="error-message">
           <FormattedMessage
             id="results-error.message"
-            defaultMessage="We're sorry. We are having some trouble completing your request. Please make sure you have completed all of the questions on the screen and try again. If you are still unable to load your benefits results, please contact myfriendben@garycommunity.org."
+            defaultMessage="We're sorry. We are having some trouble completing your request. Please make sure you have completed all of the questions on the screen and try again. If you are still unable to load your benefits results, please contact "
           />
+          {email}.
```

Comment:
Can we remove the `.`? Hard coding the `.` does not translate into every language. Also, Someone might try to email the email address plus the `.` at the end.

```suggestion
          {email}
```